src/styles/main.scss

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
```

Comment:
Ensure that the variable `$gap` is defined in your SCSS files. If it's not defined, it will cause a compilation error.